plugins/base/src/test/kotlin/superFields/PsiSuperFieldsTest.kt

```diff
@@ -0,0 +1,142 @@
+package superFields
+
+import org.jetbrains.dokka.base.testApi.testRunner.BaseAbstractTest
+import org.jetbrains.dokka.links.DRI
+import org.jetbrains.dokka.model.Annotations
+import org.jetbrains.dokka.model.InheritedMember
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Assertions.assertNull
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+
+class PsiSuperFieldsTest : BaseAbstractTest() {
+
+    @Disabled // TODO: Remove with Kotlin 1.6.20
```

Comment:
We've updated to 1.6.20-M1, but not everything that will go into the release is in M1, I think. You think this can be removed now? Or are we waiting for some specific task to be released in the compiler?

Reply:
We should check if tests are passing without flakiness

Reply:
I've checked that and it still keeps failing sometimes. We should keep them disabled